src/pages/Models/Models.test.js

```diff
@@ -36,4 +37,28 @@ describe("Models page", () => {
       "17 models: 2 blocked, 4 alerts, 11 running"
     );
   });
+
+  it("displays correct grouping view", () => {
+    global.window = { location: { pathname: "/models?groupedby=owner" } };
```

Comment:
Because you're using the `useLocation` hook you should be able to use the `MemoryRouter` here instead with an `initialEntries` defined

https://github.com/canonical-web-and-design/jaas-dashboard/blob/master/src/components/InfoPanel/InfoPanel.test.js#L19